backend/services/event_service.py

```
@@ -0,0 +1,71 @@
+from datetime import datetime
+from zoneinfo import ZoneInfo
+from fastapi import HTTPException
+from sqlalchemy.orm import Session
+from models.post import Post, Event
+from typing import Optional, Tuple
```

Ruff (0.8.2)

`typing.Tuple` is deprecated, use `tuple` instead

(UP035)